tests/testthat/test_iregenet.R

```diff
@@ -0,0 +1,13 @@
+context("iregnet prediction")
+library(iregnet)
+
+
+
+test_that("iregnet function converged", {
+  
+  load("data/H3K36me3_TDH_immune.Rdata")
```

Comment:
`data(H3K36me3_TDH_immune, package="iregnet")` which will work even if you are in the wrong directory

Reply:
RData rather than Rdata?